src/planscape/rscripts/forsys.R

```diff
@@ -331,6 +331,67 @@ get_weights <- function(priorities, configuration) {
   return(configuration$weights)
 }
 
+get_number_of_projects <- function(scenario) {
+  # this is hardcoded for now. in the future it
+  # might come from the configuration JSON
+  return(5)
+}
+
+get_min_project_area <- function(scenario) {
+  configuration <- get_configuration(scenario)
+  stand_size <- configuration$stand_size
+  min_area <- 10
```

Comment:
didn't we say LARGE was the default? shouln't we start with that and modify for medium and small?